### PR TITLE
Add support for multiple metrics files, including auto_metrics, to redcal

### PR DESCRIPTION
**Pull request by jsdillon**
With the new auto_metrics (https://github.com/HERA-Team/hera_qm/pull/391) files, it is useful to enable redcal to take these files and extract bad antennas. Ideally this will be part of the next iteration of RTP (see https://github.com/HERA-Team/hera_pipelines/issues/9).

### CODE REVIEW

**Review by dstorer — APPROVED** [review]
Looks good to me!

**Comment by jsdillon** [comment]
Thanks! I'm going to wait until the new ant_metrics is pulled in before I merge this.

**Pull request merged**
